src/server/mod.rs

```diff
@@ -221,6 +173,23 @@ async fn route(request: Request<Body>, app: Arc<App>) -> Result<Response<Body>,
             })
             .await
         }
+        (&Method::POST, "/addBatchSize") => {
```

Comment:
Would probably be useful to have an endpoint to list the current available batch sizes.

Also now I'm thinking it'd be great to have some fancier routing, maybe with `axum`? But no biggie, I can also take a stab at that in some spare moment.

Reply:
I've added the endpoint but more complex routes are out of scope for this PR.